packaging/linux/openra.6

```diff
@@ -0,0 +1,68 @@
+.TH OPENRA 6
+.SH NAME
+openra \- An open-source implementation of the Command & Conquer: Red Alert engine using .NET/Mono and OpenGL.
```

Comment:
That description was pretty much correct when it was originally written back in 2010 :)

I suggest changing this to "An open-source modernization of the early 2D Command & Conquer games." (people running the games don't need more info, and updating the repo description to the more complete "The OpenRA project recreates and modernizes the early 2D Command & Conquer games on top of a modular open source RTS game engine.  It uses .NET/Mono, SDL2, and OpenGL, and runs on Windows, Mac OS X, and Linux."